dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -3046,15 +2669,24 @@ private Map<String, Map<String, Object>> getLocalParams(ProcessDefinition proces
         return localUserDefParams;
     }
 
-    /**
-     * delete other relation
-     * @param project
-     * @param result
-     * @param processDefinition
-     */
-    @Override
-    public void deleteOtherRelation(Project project, Map<String, Object> result, ProcessDefinition processDefinition) {
-
+    private void checkWorkflowIsNotEmpty(Long workflowDefinitionCode) {
```

Comment:
NIP, I think it's better to check DAG dto.

Reply:
I added comment on this method, and will use another pr to add a public method for check if the dag is validated.